ansible/devutil/ssh_session_repo.py

```diff
@@ -348,8 +376,8 @@ def __init__(self, target, ssh_config_params, console_ssh_config_params):
             self.ssh_config = sshconf.read_ssh_config(self.target)
 
         # Add SSH config parameters
-        self.ssh_config_params = ssh_config_params
-        self.console_ssh_config_params = console_ssh_config_params
+        self.ssh_config_params = ssh_config_params if ssh_config_params is not None else {}
+        self.console_ssh_config_params = console_ssh_config_params if console_ssh_config_params is not None else {}
```

Comment:
Is the `is not None` part needed here, or can it be dropped?

Reply:
yea.. if this is dropped, the script will throw if ssh config param and console config params are not provided, so I added them to get the exception fixed.